### DISCUSSION TITLE
Awesome

### DISCUSSION THREAD

**Discussion by jakew009** [bluenviron/mediamtx#506] [General]
Just wanted to say thank you to @aler9 for such a brilliant project.

It's rare that you can download something, and that it immediately 'just works' exactly as you expect it to.

The new API is just the icing on the cake. This has saved us hundreds of hours of development time :100:

**Comment by aler9** [comment]
Thank you very much, the aim is creating a solid, community-driven component that can be used as building block for SaaS or even embedded systems. The project was born because live555 proxy server required a huge quantity of resources and wasn't fit for unmanned vehicles (which i was working on at the time) nor industrial plants (the same). The same can be said for Wowza. The problem is now solved and saved resources can be used for more useful aims.

**Reply by jakew009** [reply]
We are running it on a fleet of Raspberry Pi compute modules running off solar panels on remote CCTV systems :) Wowza, Live555, AntMedia etc are totally unsuitable for such uses.

[a comment by an automated account is not shown]
